app/src/main/res/layout/fragment_map.xml

```diff
@@ -11,6 +11,38 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:id="@+id/map"/>
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginVertical="10dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginRight="0dp"
+        android:layout_marginLeft="0dp"
+        android:elevation="10dp"
+        android:background="#fff"
+        android:id="@+id/relLayout1">
+
+        <ImageView
+            android:id="@+id/ic_magnify"
+            android:layout_width="wrap_content"
+            android:layout_height="40dp"
+            android:layout_centerVertical="true"
+            android:layout_marginLeft="5dp"
+            android:src="@drawable/ic_magnify"
+            />
+        <EditText
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_toRightOf="@id/ic_magnify"
+            android:layout_centerVertical="true"
+            android:textSize="15sp"
+            android:textColor="#000"
```

Comment:
Please replace `android:textColor="#000"` with `android:textColor="@color/black"`

Reply:
why?

Reply:
In `res/values/colors.xml`, that colour has already been set up. Changing it to the colour resource now makes it easier to implement night mode later on.